### ISSUE TITLE
memfd use by QtQml causes false positives

### ISSUE THREAD

**Issue by kushal-kumaran** [liske/needrestart#247]
With new enough kernel/plasmashell/KDE/Qt needrestart reports false-positives on memfd files used by Qt applications:

```
# needrestart   
Scanning processes...                                                                                                                                                                        
Scanning candidates...                                                                                                                                                                       
Scanning processor microcode...                                                                                                                                                              
Scanning linux images...                                                                                                                                                                     

Running kernel seems to be up-to-date.

The processor microcode seems to be up-to-date.

No services need to be restarted.

No containers need to be restarted.

User sessions running outdated binaries:
 kushal @ session #3: ksmserver[2053], plasmashell[2060]
```

```
[main] #2060 uses deleted /memfd:JITCode:QtQml
[main] #2060 is not a child
[main] #2166 uses deleted /memfd:JITCode:QtQml
[main] #2166 is a child of #2053
[main] #2053 exe => /usr/bin/ksmserver
[main] #2053 part of user session: uid=1000 sess=3
[main] #2060 exe => /usr/bin/plasmashell
[main] #2060 part of user session: uid=1000 sess=3
```

I've added `qr(^/memfd:JITCode:QtQml),` to ignore these for now, but perhaps a more general ignore pattern for all memfd might be appropriate.

**Comment by rwhector** [comment]
Is anything starting with /memfd: relevant?

**Comment by liske** [comment]
The [config file shipped with needrestart 3.6](https://github.com/liske/needrestart/blob/v3.6/ex/needrestart.conf#L156-L157) already ignores those mappings.
